app/models/ability.rb

```diff
@@ -59,6 +61,7 @@ def initialize(user)
       # Organizers can view, edit and print Applications, view participants for, view and upload materials for, print agreement letters for and manage Events
       can [:view_applicants, :edit_applicants, :view_participants, :print_applications, :manage, :view_material, :upload_material, :print_agreement_letters], Event
       can :manage, Request
+      cannot [:apply], Event
```

Comment:
I think we dont need this line, it should already be inherited from coach access rights